packages/zoe/src/instanceRecordStorage.js

```diff
@@ -47,6 +47,7 @@ export const makeInstanceRecordStorage = baggage => {
     baggage,
     'InstanceRecord',
     InstanceRecordI,
+    /** @type {(ir: InstanceRecord) => {instanceRecord: InstanceRecord}} */
```

Comment:
shouldn't `ir` (the parameter name in the parameter list in the function type) match `record` (the parameter name in the function)?

Reply:
not necessarily but it's a good practice. I'll revise